### PR TITLE
[New] Add gitalk w/ npm auto-update

### PR DESCRIPTION
**Pull request by extend1994**
Pull request for issue: #11668

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/gitalk/gitalk
 * Official website (optional, not the repository): https://gitalk.github.io/
 * NPM package url (optional): https://www.npmjs.com/package/gitalk
 * License and its reference: [MIT](https://github.com/gitalk/gitalk/blob/master/LICENSE)
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 17
   - Count of stars: 620
   - Count of forks: 49
 * NPM download stats (optional):
   - Downloads in the last day: 3
   - Downloads in the last week: 66
   - Downloads in the last month: 288

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 800 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by extend1994** [comment]
Use git auto-update rather than npm because there are more versions if beta versions are excluded.

File: ajax/libs/gitalk/package.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{
+  "name": "gitalk",
+  "filename": "gitalk.min.js",
+  "version": "1.1.4",
+  "description": "A comment plugin base on Github issues",
```
Comment (sufuf3) [inline ajax/libs/gitalk/package.json]: Maybe use [`Gitalk is a modern comment component based on Github Issue and Preact.`](https://github.com/gitalk/gitalk#gitalk) is better than `A comment plugin base on Github issues`.
What do you think?

Thanks

**Comment by extend1994** [comment]
@sufuf3 PR updated, please help review it, thanks.

**Comment by sufuf3** [comment]
Would you mind to update the PR title?
from `[New] Add gitalk@1.2.2 w/ npm auto-update` to `[New] Add gitalk w/ npm auto-update`
Because you add two versions of the lib.
What do you think?
Thank you.

**Comment by extend1994** [comment]
@sufuf3 Updated.

**Pull request merged**